plugins/woocommerce-blocks/assets/js/blocks-registry/product-collection/register-product-collection.tsx

```diff
@@ -5,17 +5,21 @@ import { BlockVariation, registerBlockVariation } from '@wordpress/blocks';
 import { addFilter } from '@wordpress/hooks';
 import { EditorBlock } from '@woocommerce/types';
 import type { ElementType } from '@wordpress/element';
-import type { BlockEditProps } from '@wordpress/blocks';
+import type { BlockEditProps, BlockAttributes } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
-import blockJson from '../block.json';
 import {
 	SetPreviewState,
 	PreviewState,
 	ProductCollectionAttributes,
-} from '../types';
+} from '../../blocks/product-collection/types';
+import {
+	DEFAULT_ATTRIBUTES,
+	INNER_BLOCKS_TEMPLATE,
+	PRODUCT_COLLECTION_BLOCK_NAME as BLOCK_NAME,
+} from '../../blocks/product-collection/constants';
```

Comment:
Nit: you can use an alias since the file is in a different scope now:

```suggestion
} from '@woocommerce/blocks/product-collection/types';
import {
	DEFAULT_ATTRIBUTES,
	INNER_BLOCKS_TEMPLATE,
	PRODUCT_COLLECTION_BLOCK_NAME as BLOCK_NAME,
} from '@woocommerce/blocks/product-collection/constants';
```

Reply:
Good catch! Made the changes in [f3a28d5](https://github.com/woocommerce/woocommerce/pull/48141/commits/f3a28d542b604974d7754c3fd2e74b182a89aa70)